src/EnergyPlus/MixedAir.cc

```diff
@@ -3204,6 +3204,9 @@ namespace MixedAir {
 		// without AirloopHVAC objects).
 		if ( InitOAControllerSetUpAirLoopHVACVariables ) {
 			if ( NumPrimaryAirSys > 0 ) {
+				if ( !allocated( AirLoopZoneInfo ) ){
+					AirLoopZoneInfo.allocate( NumPrimaryAirSys );
```

Comment:
This array is allocated at line 1524 in GetOAControllerInputs. This code is in InitOAController. So why would this code get called before this array is allocated??